File: projects/order-creation-queries/core/src/main/java/com/robinfood/core/dtos/ordercategories/OrderCategoryDTO.java
package com.robinfood.core.dtos.ordercategories;

import com.fasterxml.jackson.annotation.JsonPropertyOrder;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@AllArgsConstructor
@Builder
@Getter
public class OrderCategoryDTO {

    public Double compensation;

    public Double discounts;

    public Double grossValue;

    public Integer id;

    public String name;

    public Double netValue;

    public Double taxes;

}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/dtos/dailyreportvoucher/OrderCategoryDTO.kt
package com.robinfood.core.dtos.dailyreportvoucher

data class OrderCategoryDTO(
    val id: Long?,
    val name: String,
    val compensation: Double,
    val grossValue: Double,
    val discounts: Double,
    val taxes: Double,
    val netValue: Double
)